clusters/app.ci/openshift-user-workload-monitoring/mixins/_prometheus/prow_alerts.libsonnet

```diff
@@ -4,6 +4,17 @@
       {
         name: 'prow',
         rules: [
+          {
+            alert: 'prow-pod-crashlooping',
+            expr: 'rate(kube_pod_container_status_restarts_total{namespace="ci",job="kube-state-metrics"}[15m]) * 60 * 5 > 0',
```

Comment:
No that the label in the query was `namespace=~"ci|prow-monitoring"` and is `namespace="ci"` because
- It is a restriction of user-defined alerts.
- The namespace `prow-monitoring` does not exist any more.